event_contact/models/event.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 OpenSynergy Indonesia
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import api, fields, models
+
+
+class EventEvent(models.Model):
+    _inherit = 'event.event'
+
+    contact_ids = fields.Many2many(
+        string='Contacts',
+        comodel_name='res.partner',
+        help='Partners available to attend attendees requests for this event.')
+
+    @api.multi
+    @api.onchange("event_type_id")
+    def _onchange_type_set_contact_ids(self):
+        if self.event_type_id.contact_ids and not self.contact_ids:
+            self.contact_ids = self.event_type_id.contact_ids
```

Comment:
Why to check if there are previous contacts assigned to the `event`? Isn't it better to add those on the `event_type` to the existing ones?